CONTRIBUTING.md: fix typo & add clonning section (#114)
FerretDB/FerretDB: CONTRIBUTING.md

go version devel go1.18-[...]
```
**Note:** If using gotip and `go version` does not return version 1.18, you may need to symbolic link `gotip` to `go`.
```sh
$ ln -s `which gotip` /user/local/bin/go
```
Use `which gotip` to get the path to gotip.

> ## Clonning

Reviewer: ```suggestion
## Cloning the Repository
```

> After forking FerretDB you can clone for best practices use the following instruction:

Reviewer: ```suggestion
After forking FerretDB you can clone the repository - for rgw best practices use the following instructions:
```

Just slight grammar fix, hope you don't mind :)

Author: Sure, thanks! :)

> After forking FerretDB you can clone the repository - for rgw best practices use the following instructions:

Reviewer: ```suggestion
After forking FerretDB you can clone the repository - for the best practices use the following instructions:
```

@ekalinin I typed in “the" wrong 😄
